### PR TITLE
Rm-302

### PR DESCRIPTION
**Pull request by igorya7v**
The latest version after the close channels changes. With some minor fixes and cosmetic changes.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,7 +79,7 @@ dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.3.4'
     compile 'com.google.guava:guava:23.0'
-    compile ('io.left.rightmesh:rightmesh-java-library-dev:0.4.2-IGOR-SNAPSHOT')
+    compile ('io.left.rightmesh:rightmesh-java-library:0.4.2')
```
Comment (FrazerSeymour) [inline build.gradle]: We have a `0.4.3` now. Would it make sense to hold off until the merge into the library when we'll bump to `1.0`?

Comment (igorya7v) [reply build.gradle]: make sense

Comment (FrazerSeymour) [reply build.gradle]: Speaking of - has the library side of this stuff merged in already?

File: src/main/java/ether/Settings.java [hunk 2]
```diff
@@ -60,8 +64,9 @@
         try {
             Object obj = parser.parse(new FileReader("rm-ethereum.conf"));
             JSONObject jsonObject = (JSONObject) obj;
-            appendingZerosForETH = ((String) jsonObject.get("appendingZerosForETH"));
-            appendingZerosForTKN = ((String) jsonObject.get("appendingZerosForTKN"));
+
+            //appendingZerosForETH = ((String) jsonObject.get("appendingZerosForETH"));
+            //appendingZerosForTKN = ((String) jsonObject.get("appendingZerosForTKN"));
```
Comment (FrazerSeymour) [inline src/main/java/ether/Settings.java]: Can these just be deleted or do they need to stick around?

**Review by FrazerSeymour — APPROVED** [review]
Nitpicks aside, looks fine.

**Review by fractalic — APPROVED** [review]

File: build.gradle [hunk 3]
```diff
@@ -79,7 +79,7 @@ dependencies {
     // This dependency is found on compile classpath of this component and consumers.
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.3.4'
     compile 'com.google.guava:guava:23.0'
```
Comment (ConnorHeidema) [inline build.gradle]: If possible try switching these compiles to implementations, it provides better encapsulation but more importantly runs faster on compile. https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle

File: build.gradle (hunk 1 above)
Comment (compscidr) [reply build.gradle]: Yep sounds like it has - we can bump to 0.4.3 here.

**Review by compscidr — APPROVED** [review]
Looks good - just the minor comments others made and we are ready to merge.

File: build.gradle (hunk 1 above)
Comment (FrazerSeymour) [reply build.gradle]: Okay. Somehow I had rattling around in memory that we were bumping the library to 1.0 once payment channels were in. :upside_down_face: So just 0.4.3 then.

File: src/main/java/ether/Settings.java (hunk 2 above)
Comment (igorya7v) [reply src/main/java/ether/Settings.java]: Uncommented, maybe we will need for testing.

File: build.gradle (hunk 3 above)
Comment (igorya7v) [reply build.gradle]: done

**Pull request merged**